Frameworks/CoreText/CTRun.mm

```diff
@@ -130,7 +135,7 @@ void CTRunGetPositions(CTRunRef run, CFRange runRange, CGPoint* outPositions) {
 }
 
 /**
- @Status Interoperable
+ @Status Stub
```

Comment:
Correct

---
In reply to: [77757935](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77757935) [](ancestors = 77757935)